jobs/integr8ly/installation-pipeline.yaml

```diff
@@ -154,8 +167,17 @@
                             """
                         } // dir
                     } // stage
-                } // node
 
+                     stage('Execute "After Installation" Workarounds') {
+                        if (BRANCH == 'master') {
+                            // this patches WALKTHROUGHS_LOCATIONS to use master tag
+                            sh """
+                                oc login https://master.${YOURCITY}.openshiftworkshop.com -u admin@example.com -p Password1 --insecure-skip-tls-verify
```

Comment:
Is this required? This runs directly on bastion host so we should be logged as `system:admin` already, should not we?

Reply:
Yes it is likely unneccessary.